src/query/generated/assets/openapi/spec.yml

```diff
@@ -600,6 +600,58 @@ paths:
           description: ""
           schema:
             $ref: "#/definitions/GenericError"
+  /services/m3db/database/config/bootstrappers:
+    post:
+      tags:
+      - "M3DB Database"
+      - "M3DB"
+      summary: "Configure M3DB set bootstrappers dynamically, which override config"
```

Comment:
"Dynamically override the bootstrappers configured in M3DBs node-level configuration"